src/cargo/core/features.rs

```diff
@@ -904,7 +906,7 @@ impl CliUnstable {
                 self.features = Some(feats);
             }
             "separate-nightlies" => self.separate_nightlies = parse_empty(k, v)?,
-            "multitarget" => self.multitarget = parse_empty(k, v)?,
+            "multitarget" => stabilized_warn(k, "1.63", STABILISED_MULTITARGET),
```

Comment:
I'm terrible about tracking the releases and mapping that to master.

iirc we are coming up on 1.62 being public.  Isn't 1.63 then a little soon?

Reply:
`1.6.2` [releases June 30th](https://forge.rust-lang.org/#current-release-versions) and [No tool breakage week](https://forge.rust-lang.org/#no-tools-breakage-week) starts June 22nd.

Reply:
I need to watch that more closely.

However, saw this fun bit

>  (Cargo is not subject to the toolstate system and instead just has to always work.)

https://forge.rust-lang.org/infra/toolstate.html

Reply:
Fine to postpone if we think it is too rush to stabilize, though I feel like it has lingered on nightly for a long long time.

Reply:
My concern is more so whether we specified the right stablization target for when current master will be on stable.